submitted_models/emesent_hovermap_sensor_config_1/specifications.md

```diff
@@ -72,13 +72,13 @@ Based on the tests specified in the DARPA SubT Challenge [Model Preparation Guid
 
 * _x_ linear velocity range from -5.0 m/s to 5.0 m/s
 * _y_ linear velocity range from -5.0 m/s to 5.0 m/s
-* _z_ linear velocity range from -2.8 m/s to 2.8 m/s
-* _x_ linear acceleration range from -4.0 m/s<sup>2</sup> to 4.0 m/s<sup>2</sup>
-* _y_ linear acceleration range from -4.0 m/s<sup>2</sup> to 4.0 m/s<sup>2</sup>
-* _z_ linear acceleration range from -5.0 m/s<sup>2</sup> to 5.0 m/s<sup>2</sup>
+* _z_ linear velocity range from -3.0 m/s to 3.0 m/s
+* _x_ linear acceleration range from -2.0 m/s<sup>2</sup> to 2.0 m/s<sup>2</sup>
+* _y_ linear acceleration range from -2.0 m/s<sup>2</sup> to 2.0 m/s<sup>2</sup>
+* _z_ linear acceleration range from -9.8 m/s<sup>2</sup> to 9.8 m/s<sup>2</sup>
```

Comment:
The XYZ acceleration noted here does not match the values in `spawner.rb`.  The provided validation data supports the values in the `spawner.rb` file as opposed to the values changed here.
![xyz_acceleration](https://user-images.githubusercontent.com/61025344/114588043-3e386000-9c54-11eb-893c-124949e612f4.png)